packages/manager/modules/pci/src/projects/project/ai/apps/dashboard/general-information/general-information.html

```diff
@@ -17,6 +17,33 @@
                     data-description="{{:: $ctrl.app.name }}"
                 ></oui-tile-definition>
 
+                <oui-tile-definition
+                    data-term="{{:: 'pci_apps_general_information_info_app_image' | translate }}"
+                >
+                    <oui-tile-description>
+                        <div>
+                            <input
+                                class="oui-input"
+                                disabled
```

Comment:
**question:** If the `<input>` element is always `disabled`, couldn't we consider using `readonly` attribute instead?

Reply:
Sure, changed here: https://github.com/ovh/manager/pull/8002/commits/39a6aa39e7e6cfc18a8ee3f67f98e08358683eec